test/configCases/web/container-reference-prefetch/index.js

```diff
@@ -0,0 +1,28 @@
+// This config need to be set on initial evaluation to be effective
+__webpack_nonce__ = "nonce";
+
+it("should prefetch the remote async module", () => {
+	__non_webpack_require__("./host-entry.js");
+
+	expect(document.head._children.length).toBe(0);
+
+	return import(/* webpackChunkName: "host-entry" */ "./host-entry")
+		.then(module => module.getRemoteModule())
+		.then(module => {
+			expect(module.default).toBe('remote ./async-module');
+			expect(document.head._children.length).toBe(1);
+
+			const link = document.head._children[0];
+			expect(link._type).toBe("link");
+			expect(link.rel).toBe("prefetch");
+			expect(link.as).toBe("script");
+
+			/**
+			 * What should this be....?
+			 */
+			expect(link.href).toBe("remote/async-module.js");
```

Comment:
Because we're using `moduleScope` in the `test.config.js`, will there ever be an actual `<link>` added to the head, even if this were to be implemented in `container-reference-plugin`?

Reply:
No not this way. Adding the prefetch tag would be delegated to the container, but the container is simulated here.
It would make sense to compile a real container in a separate test case to get the full Workflow.
Check the test cases that start with numbers to see how to write two test cases that work together. E.g. 1-container-full